index.html

```diff
@@ -19,161 +19,216 @@
+                src="images/graphics/image_dots.png"
+                alt="Graphic picture"
               />
             </div>
-            <div class="project-detail">
-              <h2 class="project-name">Project name goes here</h2>
-              <ul class="languages">
-                <li>HTML/CSS</li>
-                <li>Ruby on Rails</li>
-                <li>JavaScript</li>
-              </ul>
-              <a href="" class="btn btn-pro"
-                >See this project <span class="arrow">&rarr;</span></a
-              >
-            </div>
           </div>
-          <div class="proj-card">
-            <div class="proj-img">
-              <img
-                src="images/project/Rectangle21(3).png"
-                alt="Project Picture"
-              />
-            </div>
-            <div class="project-detail">
-              <h2 class="project-name">Project name goes here</h2>
-              <ul class="languages">
-                <li>HTML/CSS</li>
-                <li>Ruby on Rails</li>
-                <li>JavaScript</li>
-              </ul>
-              <a href="" class="btn btn-pro"
-                >See this project <span class="arrow">&rarr;</span></a
-              >
-            </div>
+          <img
+            class="img-geometry-large landing-item"
+            src="images/graphics/image-geometry_large.png"
+            alt="Graphic picture"
+          />
+        </section>
+        <section class="work" id="portfolio">
+          <div class="work-title">
+            <h2 class="title">Projects</h2>
           </div>
-          <div class="proj-card">
-            <div class="proj-img">
-              <img
-                src="images/project/Rectangle21(4).png"
-                alt="Project Picture"
-              />
-            </div>
-            <div class="project-detail">
-              <h2 class="project-name">Project name goes here</h2>
-              <ul class="languages">
-                <li>HTML/CSS</li>
-                <li>Ruby on Rail</li>
-                <li>JavaScript</li>
-              </ul>
-              <a href="" class="btn btn-pro"
-                >See this project <span class="arrow">&rarr;</span></a
-              >
-            </div>
+          <img
+            class="img-geometry-dot"
+            src="images/graphics/image_dots-2.png"
+            alt="Graphic picture"
+          />
+          <img
+            class="proj-img proj-img_1"
+            src="images/project/Rectangle21.png"
+            alt="Project Picture"
+          />
+          <div class="project-detail project-detail_1">
+            <h2 class="project-name">Project name goes here</h2>
+            <ul class="languages">
+              <li>HTML/CSS</li>
+              <li>Ruby on Rails</li>
+              <li>JavaScript</li>
+            </ul>
+            <a href="" class="btn btn-pro"
+              >See this project <span class="arrow">&rarr;</span></a
+            >
           </div>
```

Comment:
- [ ] OPTIONAL: I recommend aligning the arrow in the centre vertically, right now it is located close to the top border of the button: 
<img width="122" alt="Screenshot 2022-02-18 at 13 21 17" src="https://user-images.githubusercontent.com/77038610/154654389-9406fce5-a8bd-42a3-8dee-4b83d550dd9f.png">